pkgs/development/python-modules/jsonrpcserver/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, apply_defaults
+, jsonschema
+}:
+
+buildPythonPackage rec {
+  pname = "jsonrpcserver";
+  version = "4.1.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "649680c293facb6ae7c3f5c8028e4623c55195db5216847e9f25f85cba2d443a";
+  };
+
+  propagatedBuildInputs = [ apply_defaults jsonschema ];
+
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
Indeed the `tests` folder is not included in the release tarball. However, if I checkout from GitHub, the folder is there, but I still see `Ran 0 tests in 0.000s`, please advise.

This is the spec I am using:

```
{ stdenv
, buildPythonPackage
, fetchFromGitHub
}:

buildPythonPackage rec {
  pname = "apply_defaults";
  version = "0.1.4";

  src = fetchFromGitHub {
    owner = "bcb";
    repo = pname;
    rev = version;
    sha256 = "1nn2mch5spq8mhk741ia7zkb2bgjfn7bzxpavb476jzs38pb39d5";
  };

  meta = with stdenv.lib; {
    description = "Apply values to optional params";
    homepage = "https://github.com/bcb/apply_defaults";
    license = licenses.mit;
    maintainers = with maintainers; [ prusnak ];
  };

}
```